### ISSUE TITLE
Build summary table of Results:Param field

### ISSUE THREAD

**Issue by stdavis** [agrc/ugs-db#15]
`dbseeder` should create a table that contains all of the unique values for the Results:Param field. This table will be used by the web application to provide an auto-suggest when the user is typing parameters into the parameter filter.

This table should be updated any time the data is updated.

**Comment by stdavis** [comment]
This could be a SQL view, couldn't it? Something like:
```sql
SELECT DISTINCT [Param]
FROM [UGSWaterChemistry].[dbo].[Results]
```
Or would that be hard on the database querying it every time the app loaded?

**Comment by stdavis** [comment]
After a few tests I think that the view isn't a good idea. We just need to generate a new table on each update. It shouldn't be a big deal.

**Comment by stdavis** [comment]
Closed via #33
